crates/sqllib/src/casts.rs

```diff
@@ -932,44 +918,51 @@ pub fn cast_to_s_Timestamp(value: Timestamp, size: i32, fixed: bool) -> String {
 }
 
 #[inline]
-pub fn cast_to_s_i(value: isize, size: i32, fixed: bool) -> String {
-    let result = value.to_string();
+pub fn cast_to_s_Date(value: Date, size: i32, fixed: bool) -> String {
+    let dt = value.to_date();
```

Comment:
chrono types have methods to serialize themselves in any format. Might have saved you a few lines of code.